make install build fails on non-wsl Windows 11 environment

## `make install` fails on make install fails on non-wsl Windows 11 environment

### Description

When trying to run `make install` on Windows 11 using non-wsl environment (`cmd`, `Git bash`) - build it fails with the next error:
`..\tm2\pkg\commands\utils.go:44:38: cannot use fd (variable of type syscall.Handle) as int value in argument to term.ReadPassword`

Might not need resolution since Non-WSL environment is not recommended (means, not supported?) according to README.md

### Your environment

* Windows 11 Pro 22621.2134
* master branch b40ac9cda5a5482a4bda7fe12675937a86dba69c

### Steps to reproduce

* Go to gno dir <br />
* Run `make install` <br />
* Observe an error: <br />
```
$ make install
C:/ProgramData/chocolatey/lib/make/tools/install/bin/make.exe --no-print-directory -C ./gno.land        install.gnokey
go install ./cmd/gnokey
# github.com/gnolang/gno/tm2/pkg/commands
..\tm2\pkg\commands\utils.go:44:38: cannot use fd (variable of type syscall.Handle) as int value in argument to term.ReadPassword
make[1]: *** [Makefile:22: install.gnokey] Error 1
make: *** [Makefile:13: install_gnokey] Error 2
```

--- a/tm2/pkg/commands/utils.go
+++ b/tm2/pkg/commands/utils.go
@@ -2,8 +2,8 @@ package commands
 
 import (
 	"errors"
+	"os"
 	"strings"
-	"syscall"
 
 	"golang.org/x/term"
 )
@@ -39,7 +39,7 @@ func (io *IO) readLine() (string, error) {
 // readPassword reads the password from a terminal
 // without local echo
 func readPassword() (string, error) {
-	fd := syscall.Stdin
+	fd := int(os.Stdin.Fd())
 
 	inputPass, err := term.ReadPassword(fd)
 	if err != nil {

